server/db/models/enhancement.js

```diff
@@ -0,0 +1,41 @@
+const crypto = require('crypto');
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Animal = db.define('animal', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  description: {
+    type: Sequelize.STRING,
+    allowNull: false,
+
+  },
+  imageUrl: {
```

Comment:
i think there's an isUrl validation you can add

Might also consider adding a hook to put in a placeholder image if none is provided. But maybe add an issue for that and leave it for later.